### ISSUE TITLE
Can't create a Pool in Devnet.

### ISSUE THREAD

**Issue by livcoding2** [raydium-io/raydium-frontend#21]
it is imposible to create a pool in Devnet.
Look at Wallet log:

--------------------------
`Program ID:
"ComputeBudget111111111111111111111111111111"

Data:
[3,168,97,0,0,0,0,0,0]

Program ID:
"ComputeBudget111111111111111111111111111111"

Data:
[2,192,39,9,0]

System Program - CreateAccountWithSeed
From account
SHITTOKEN
Created account
HhVFaAn9azqDL5PU1AsQNfph9nyMLZBBFmV7Ftpv2YJs
Base account
Base
SHITTOKEN
Seed
Gc4Febj6CpaG5tFLXtrj6zgGy2tyifiW
Lamports
5.00203928 SOL
Space
165
Owner programme
TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA
Token programme - InitializeAccount
Initialised account
HhVFaAn9azqDL5PU1AsQNfph9nyMLZBBFmV7Ftpv2YJs
Mint
So11111111111111111111111111111111111111112
Owner
SHITTOKEN
Rent sysvar
Address look-up table account:
2immgwYNHBbyVQKVGCEkgWpi53bLwWNRMB5G2nbgYV17
Address look-up table index:
5
Program ID:
"675kPX9MHTjS2zt1qfr1NYHuzeLXfQM9H24wFSUt1Mp8"

Accounts:
TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA
Address look-up table account:
2immgwYNHBbyVQKVGCEkgWpi53bLwWNRMB5G2nbgYV17
Address look-up table index:
7
11111111111111111111111111111111
Address look-up table account:
2immgwYNHBbyVQKVGCEkgWpi53bLwWNRMB5G2nbgYV17
Address look-up table index:
5
BpM5xz9bbcLsenX8aZsTvvA72UP1westD9u3mVRMjdLJ
5Q544fKrFoe6tsEbD7S8EmxGTJYAKtTVhAW5Q5pge4j1
9CCpiYwPJTWFKta7tfg1muomoEzzVjXeoPhJPFfMWmGJ
59vLkwJJGexijHpN1bbfiMdbtpW8aP7CQUksEwJGnB3r
E6byCwMr25BYhDDf25CaB42HdHsssY7eKhZ9sf8SSPF7
So11111111111111111111111111111111111111112
3pc5SYdSh26xmtUgU9USqz4cmWo9qZxdFtXzHSTqkbUh
2JF55pirJNUYZQiYd3t4nSneQXqcs12WEy5B2feNSvbr
JCtPkBQuP1sTcktysoqtLBYde8XtcSarDfHi1upHB6bG
9DCxsMizn3H1hprZ7xWe6LDzeUeZBksYFpBWBtSf1PQX
7YttLkHDoNj9wyDur5pM1ejNaAvT9X4eqaYcHQqtj2G5
Address look-up table account:
2immgwYNHBbyVQKVGCEkgWpi53bLwWNRMB5G2nbgYV17
Address look-up table index:
11
HBxX7G7xiwh1KeW7ebXEoWF6HcLPRedncET8v4pTD7cT
SHITTOKEN
D83QpRP5uucXZ96Xfgn64Ftpsy3PYRJiUHERGfp2rb3A
HhVFaAn9azqDL5PU1AsQNfph9nyMLZBBFmV7Ftpv2YJs
DhHKqo4NchKrcVS5Sz3PPeTZW97BrwSBvDoNret1V4b7
Data:
[1,254,182,208,244,101,0,0,0,0,0,242,5,42,1,0,0,0,0,0,100,167,179,182,224,13]

Token programme - CloseAccount
Closed account
HhVFaAn9azqDL5PU1AsQNfph9nyMLZBBFmV7Ftpv2YJs
To account
SHITTOKEN
Owner
SHITTOKEN
-----------------------------------

Program ID:
"**675kPX9MHTjS2zt1qfr1NYHuzeLXfQM9H24wFSUt1Mp8**" this is for mainet.

https://docs.raydium.io/raydium/protocol/developers/addresses

any help is appreciated!

**Comment by AkshatGiri** [comment]
Same issue. Tried rolling back to older commits but no luck.

**Comment by JuanRdBO** [comment]
The issue lies in this file: https://github.com/raydium-io/raydium-sdk/blob/master/src/base/lookupTableCache.ts

When the new transaction to create a market is called [here](https://github.com/raydium-io/raydium-frontend/blob/e0a65b860f7ce65d044be295a98f2165f5381fc2/src/application/txTools/handleTx/createVersionedTransaction.ts#L20C25-L20C43), it creates a tx with the mainnet lookup table instead of the devnet one. 

If you comment that line out it will work :-)

**Comment by livcoding2** [comment]
> The issue lies in this file: https://github.com/raydium-io/raydium-sdk/blob/master/src/base/lookupTableCache.ts
> 
> When the new transaction to create a market is called [here](https://github.com/raydium-io/raydium-frontend/blob/e0a65b860f7ce65d044be295a98f2165f5381fc2/src/application/txTools/handleTx/createVersionedTransaction.ts#L20C25-L20C43), it creates a tx with the mainnet lookup table instead of the devnet one.
> 
> If you comment that line out it will work :-)

Hello, and thnak you for your time and response.
First of all, forgive my ignorance and lack of skill, I'm at the beginning.
I've been trying for about 3 hours to comment a line and it doesn't work. I guess that's what it's about.

`const spawnedTransactions = await buildSimpleTransaction({
    connection,
    payer: wallet,
    innerTransactions: transactions,
    makeTxVersion: txVersion,
    //addLookupTableInfo: LOOKUP_TABLE_CACHE,`
 ?

Thank you!

**Comment by livcoding2** [comment]
Hello, and thank you for your time and response.
First of all, forgive my ignorance and lack of skill, I'm at the beginning.
I've been trying for about 3 hours to comment a line and it doesn't work. I guess that's what it's about.

<img width="544" alt="Screenshot 2024-03-19 at 11 26 16" src="https://github.com/raydium-io/raydium-frontend/assets/119439757/5976ce01-df48-4421-b1fc-3761718018a3">


?

Thank you!

**Comment by JuanRdBO** [comment]
Yeah, no worries. You have to make sure that line is not read by the code so you just comment or delete the line

**Comment by jeremybeal11** [comment]
I'm having similar issues as @livcoding2 . I deleted the addLookupTableInfo, and it's still not connecting to devnet. Any luck for anyone?

**Comment by Brovmos** [comment]
@AkshatGiri @jeremybeal11
